app/models/badges/v3_pioneer_badge.rb

```diff
@@ -0,0 +1,186 @@
+module Badges
+  class V3PioneerBadge < Badge
+    seed "V3 Pioneer",
+      :ultimate,
+      'v3-pioneer',
+      'Awarded for having helped build Exercism version 3'
+
+    def award_to?(user)
+      V3_PIONEERS.include?(user.handle)
+    end
+
+    def send_email_on_acquisition?
+      false
```

Comment:
I actually think we should email these people about it. So set this to `true` and use `send_email: true` in the script?

Reply:
Sure. `send_email: true` is the default, so we can omit it.